lib/utils/AddableProductFields.dart

```diff
@@ -0,0 +1,17 @@
+/// Fields of a [Product] that can be simply added.
+enum AddableProductField {
+  BRANDS,
+  STORES,
+  COUNTRIES,
+}
+
+extension AddableProductFieldExtension on AddableProductField {
+  static const Map<AddableProductField, String> _KEYS = {
+    AddableProductField.BRANDS: 'add_brands',
+    AddableProductField.STORES: 'add_stores',
+    AddableProductField.COUNTRIES: 'add_countries',
+  };
+
+  /// Returns the key of the product field
+  String get addableKey => _KEYS[this] ?? '';
```

Comment:
Instead of an empty string, an Exception would be better, to let us know a case is missing